source/opt/set_spec_constant_default_value_pass.cpp

```diff
@@ -248,5 +250,93 @@ bool SetSpecConstantDefaultValuePass::Process(ir::Module* module) {
   return modified;
 }
 
+// Returns true if the char pointed by the given pointer is '\0' or '}'.
+bool IsEnd(const char* const ptr) {
+  if (*ptr == '\0' || *ptr == '}') {
+    return true;
+  }
+  return false;
+}
+
+// Returns true if the char pointed by the given pointer is ' ', '\n', '\t',
+// '\r'.
+bool IsSpace(const char* const ptr) {
```

Comment:
`std::isspace`? But I kinda remember there are some pitfalls of using that? @dneto0